Mathlib/Data/Finset/Basic.lean

```diff
@@ -807,6 +807,11 @@ theorem ssubset_iff_exists_cons_subset : s ⊂ t ↔ ∃ (a : _) (h : a ∉ s),
   obtain ⟨a, hs, ht⟩ := not_subset.1 h.2
   exact ⟨a, ht, cons_subset.2 ⟨hs, h.subset⟩⟩
 
+theorem cons_swap (ha : a ∉ s) (hb : b ∉ s.cons a ha) :
+    (s.cons a ha).cons b hb = (s.cons b fun h ↦ hb (mem_cons.mpr (.inr h))).cons a fun h ↦
```

Comment:
As mentioned elsewhere, I think this should really say `(s.cons b hb).cons a ha` aka `cons a (cons b hb) ha`, which matches the order of `Multiset.cons_swap`.